frameworks/CSharp/appmpower/src/appMpower.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>

```

Comment:
Can run `net6.0` instead of `net5.0`; just 'cos the "official" benchmarks are laggards on version doesn't mean you need to be 😎

Reply:
Follow your point, but would first like to compare apples with apples: my interest is to see how the odbc compares to the "default" npgsql implementations in the official benchmarks. As these are version 5, like to keep it like that as a start. Will afterwards soon move to 6. Another beginner's question: if I "Resolve conversation", does this mean I accept the proposed change, or does it just close this comment ?

Reply:
> if I "Resolve conversation", does this mean I accept the proposed change, or does it just close this comment ?

Just closes comment